### PR TITLE
(jobs) support `pipeline` in jobs response

### PR DESCRIPTION
**Pull request by zaquestion**
Ref: https://docs.gitlab.com/ce/api/jobs.html#get-a-single-job

Side note: tested this locally and with
https://github.com/zaquestion/lab/pull/273 with success

### CODE REVIEW

File: jobs.go [hunk 1]
```diff
@@ -46,8 +46,14 @@ type Job struct {
 	FinishedAt *time.Time `json:"finished_at"`
 	ID         int        `json:"id"`
 	Name       string     `json:"name"`
-	Ref        string     `json:"ref"`
-	Runner     struct {
+	Pipeline   struct {
+		ID     int64  `json:"id"`
```
[the remark this answers is not in the document]
Comment (zaquestion) [reply jobs.go]: Nope, I generated that struct from the api response and didn't give it a second thought. Updated to just be an `int`

**Comment by svanharmelen** [comment]
LGTM! Thanks!

**Comment by zaquestion** [comment]
@svanharmelen could you possibly tag this commit as a patch release so I can include it in my project. Unless I'm missing something with modules... I think I have to rely on dependencies tagging commits to get use them now ??

**Comment by zaquestion** [comment]
Ah, so I was missing something and naturally I found it 4m after throwing my hands in the air...
```
GO111MODULE=on go mod edit -require github.com/xanzy/go-gitlab@master
```
Allows me to use go-gitlab on the master branch :D

**Comment by svanharmelen** [comment]
Yeah, you can indeed specify master or a specific commit as well. I’ll tag it as a new release when I’m behind my mac again. But that will not be today...

**Comment by svanharmelen** [comment]
@zaquestion just tagged the latest commit...

**Pull request merged**